### PR TITLE
feat: update hermes binaries to `1.7.3` and add a script to download binaries

### PR DESCRIPTION
**Pull request by Pantani**

### CODE REVIEW

File: scripts/gen-hermes [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+REPO_URL="https://github.com/informalsystems/hermes"
+DEST_FOLDER="hermes"
+
+# Get the latest release version from GitHub API
+RELEASE_INFO=$(curl -s -L "$LAST_RELEASE_API")
+LATEST_TAG=$(echo "$RELEASE_INFO" | grep -o '"tag_name": "[^"]*' | grep -o '[^"]*$')
+echo "Download Hermes $LATEST_TAG binaries"
+
+# Loop through each platform and download the corresponding binary
+#hermes-v1.7.3-x86_64-unknown-linux-gnu.zip
```
Comment (jeronimoalbi) [inline scripts/gen-hermes:14]: This may have been left out

```suggestion
```

File: .github/workflows/gen-hermes-binaries.yml
Comment (jeronimoalbi) [inline .github/workflows/gen-hermes-binaries.yml:1]: @Pantani how would we trigger these workflows when a new binary release is available? I mean, what would be the workflow when new Gex or Hermes binaries are available to make a new release from this repo?

Comment (Pantani) [reply .github/workflows/gen-hermes-binaries.yml:1]: We need to save the last download version somewhere and check new versions daily. Maybe we can open an issue for it

**Review by jeronimoalbi — APPROVED** [review]

File: .github/workflows/gen-hermes-binaries.yml
Comment (jeronimoalbi) [reply .github/workflows/gen-hermes-binaries.yml:1]: Issue #23

**Pull request merged**